optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java

```diff
@@ -204,13 +224,18 @@ void recordAndRegisterBeans(OptaPlannerRecorder recorder, RecorderContext record
                         GizmoMemberAccessorEntityEnhancer.getDroolsInitializer(recorderContext)))
                 .done());
 
+        StringWriter effectiveSolverConfigXMLWriter = new StringWriter();
+        SolverConfigIO solverConfigIO = new SolverConfigIO();
+        solverConfigIO.write(solverConfig, effectiveSolverConfigXMLWriter);
```

Comment:
Seems to me that we are duplicating this code. How about introducing a method that takes the config and returns a string?

Reply:
Look like extra/old code to me; `effectiveSolverConfigXMLWriter` is not used elsewhere